tools/build_h3_tools.sh

```diff
@@ -68,13 +72,59 @@ else
   num_threads=$(sysctl -n hw.logicalcpu)
 fi
 
+# boringssl
+echo "Building boringssl..."
+
+# We need this go version.
+if [ ! -d ${BASE}/go ]; then
+  sudo mkdir ${BASE}/go
+fi
+
+if [ `uname -m` = "arm64" ]; then
```

Comment:
In a Linux container in an m1, `uname -m` produces aarch64. Let's update this to:

```sh
if [ `uname -m` = "arm64" -o `uname -m` = "aarch64" ]; then
```